modularhistory/config/caches.py

```diff
@@ -10,24 +10,46 @@
 Cache = Dict[str, Any]
 CACHES: Dict[str, Cache]
 
+redisCache = {
```

Comment:
nitpick:  Python convention is for constants to be in all caps -->
`REDIS_CACHE = {`

Reply:
(And for other variable names, the JavaScript convention is camel case, but the Python convention is snake case.)